browse/templates/base.html

```diff
@@ -35,6 +34,7 @@
   {%- include "feedback_collector_js.html" -%}
   {% if config['BROWSE_USER_BANNER_ENABLED'] %}
   <link rel="stylesheet" type="text/css" media="screen" href="//static.arxiv.org/css/user-banner.css?v=1.0" />
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
this should probably come before the if statement for user_banner? otherwise it doesn't load on pages without user_banner.

Reply:
Good catch! As you can see I was in a hurry. Fixed!